src/webplugin/popup.ts

```diff
@@ -0,0 +1,155 @@
+let dbName = "_ionicstorage";
```

Comment:
General comment: it does not appear this code is compatible with the new BigInt changes in the Core and API right?